test/plugin/scenarios/gateway-2.0.x-scenario/config/expectedData.yaml

```diff
@@ -57,7 +57,7 @@ segmentItems:
       tags:
       - {key: url, value: 'http://localhost:8080/provider/b/testcase'}
       - {key: http.method, value: GET}
-      - {key: status_code, value: '200'}
+      - {key: http.status_code, value: '200'}
```

Comment:
FYI @JaredTan95 After this change, Resin plugins in https://github.com/SkyAPM/java-plugin-extensions should break. We need a fix after 8.8.0 release, and release 3.0.0. Could you create an issue there to track this?